### ISSUE TITLE
feat[dev]: ✨ implement A11y Re-Improvements to Media Template

### ISSUE THREAD

**Issue by yoezlem** [solid-design-system/solid#2065]
## Description
This tasks describes the Figma implementation for: #2048 

This task describes the Storybook implementation for the Media Template as part of the A11y re-improvements outlined in Epic #2048.

A11y improvements for Media Template can be found here: [Media Template Notes](https://www.notion.so/matuzo/Media-1baf3f6e1bc08166b1c7ed12546acb3a).

------------------------------------------

## Comment
**Problem:** The copyright text is hard to read. Tools don’t/can’t provide a ratio when testing against background images, but technically, it’s a violation of [[WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=143#contrast-minimum)](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=143#contrast-minimum).
**Solution:** Consider using background colors for copyright text.

## DoR
- [x] Item has been estimated by the team
- [x] Item dependencies have been identified and documented

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch
